CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## v1.1.3
```

Comment:
Last fix, just need this version to match the one in the pack.yaml then we're all good!